src/neo/SmartContract/JsonSerializer.cs

```diff
@@ -48,6 +48,11 @@ public static JObject Serialize(StackItem item)
 
                         foreach (var entry in map)
                         {
+                            if (!(entry.Key is ByteString))
```

Comment:
I think this should be done in `SerializeToByteArray()`, as it is used by `System.Json.Serialize`.

Reply:
Fixed.